test/csv/test_csv_filter.rb

```diff
@@ -78,6 +78,17 @@ def test_option_input_col_sep
                  run_csv_filter(csv, "--input-col-sep=:"))
   end
 
+  def test_option_input_quote_char
+    input_quote_char = "'"
+    str = CSV.generate(quote_char: input_quote_char) do |csv|
```

Comment:
Can we use better variable name?

```suggestion
    csv_text = CSV.generate(quote_char: input_quote_char) do |csv|
```